<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndoorPositioningSystemUI</class>
 <widget class="QMainWindow" name="IndoorPositioningSystemUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1708</width>
    <height>971</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>2136</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IndoorPositioningSystem</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Play_Pause">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>880</y>
      <width>41</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Duration">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>840</y>
      <width>1141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_Current_Time">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>840</y>
      <width>67</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Total_Time">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>840</y>
      <width>67</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Video">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>1281</width>
      <height>721</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Video_Frame_ID">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Video_Frame_Timestamp">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>32</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame Timestamp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_ID_1">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>64</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_timestamp_1">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>86</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Timestamp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor101_1">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>114</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Anchor 101:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor102_1">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>144</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Anchor 102:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Video_Frame_ID_value">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>32</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Video_Frame_Timestamp_value">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>33</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_ID_value_1">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>64</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_timestamp_value_1">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>87</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor101_value_1">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>114</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor102_value_1">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>144</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_ID_value_2">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>216</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor102_2">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>296</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Anchor 102:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_ID_2">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>216</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor101_value_2">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>266</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor102_value_2">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>296</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor101_2">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>266</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Anchor 101:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_timestamp_2">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>238</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Timestamp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_timestamp_value_2">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>241</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_ID_value_3">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>366</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor102_3">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>446</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Anchor 102:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_ID_3">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>366</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor101_value_3">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>416</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor102_value_3">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>446</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor101_3">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>416</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Anchor 101:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_timestamp_3">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>388</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Timestamp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_timestamp_value_3">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>392</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_UWB_Data_Analysis">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>490</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyse UWB Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Video_Stream">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 13pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Video Stream:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>20</y>
      <width>21</width>
      <height>851</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_Video_Stream_2">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>30</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 13pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>UWB Data Stream:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_UWB_Localization">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>840</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>UWB Localization Schema</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Export_Data">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>880</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Data for Model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Predict_Model">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>880</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Predict by model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Predict_Optical">
    <property name="geometry">
     <rect>
      <x>1048</x>
      <y>880</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Predict by height</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_measurement_time_1">
    <property name="geometry">
     <rect>
      <x>1336</x>
      <y>170</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Measurement time (ms):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_measurement_time_value_1">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>172</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_measurement_time_2">
    <property name="geometry">
     <rect>
      <x>1336</x>
      <y>320</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Measurement time (ms):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_measurement_time_value_2">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>322</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_measurement_time_3">
    <property name="geometry">
     <rect>
      <x>1336</x>
      <y>470</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Measurement time (ms):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_measurement_time_value_3">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>472</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1708</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Video"/>
    <addaction name="actionLoad_model_params"/>
    <addaction name="actionLoad_intrinsic_calibration_params"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Video">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOpen_UWB_data">
   <property name="text">
    <string>Open UWB data</string>
   </property>
  </action>
  <action name="actionLoad_model_params">
   <property name="text">
    <string>Load model params</string>
   </property>
  </action>
  <action name="actionLoad_intrinsic_calibration_params">
   <property name="text">
    <string>Load camera intrinsic params</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
